CHANGELOG.md

```diff
@@ -2,7 +2,8 @@
 
 ## Unversioned
 
-- Major: Added support for Twitch's Chat Replies. [Wiki Page](https://wiki.chatterino.com/Features/#message-replies) (#3722, #3989, #4041)
+- Major: Added support for Twitch's Chat Replies. [Wiki Page](https://wiki.chatterino.com/Features/#message-replies) (
+  #3722, #3989, #4041, #4047)
 - Major: Added multi-channel searching to search dialog via keyboard shortcut. (Ctrl+Shift+F by default) (#3694, #3875)
```

Comment:
```suggestion
- Major: Added multi-channel searching to search dialog via keyboard shortcut. (Ctrl+Shift+F by default) (#3694, #3875)
- Minor: Added a setting to hide the leading `@mention` in reply threads. (Enabled by default) (#4047)
- Minor: Added `Participated Reply Threads` highlight for reply threads that you have participated in. (#4047)
```

I'm just spitballing ideas here, this can be discussed and applied during review.

Reply:
IMO the inital change is fine, minus the newline in the changelog entry.